soc/nordic/common/Kconfig

```diff
@@ -4,6 +4,9 @@
 config HAS_NORDIC_DMM
 	bool
 
+config HAS_NORDIC_RAM_CTRL
+	bool "nRF RAM control helper support"
```

Comment:
Why is this prompt added? If I understand it correctly, this symbol is supposed to be selected by SoCs, not configured by users.

Reply:
removed prompt